cluster/gce/gci/configure.sh

```diff
@@ -448,11 +505,17 @@ function ensure-container-runtime {
     fi
     docker version
   elif [[ "${container_runtime}" == "containerd" ]]; then
+    set -x
     if ! command -v ctr >/dev/null 2>&1; then
-      echo "ERROR ctr not found. Aborting."
-      exit 2
+      install-containerd
```

Comment:
name this so it is clear that it may be a no-op, and is ubuntu-only?

Reply:
why do we only call install-containerd if `ctr` can't be found?

Reply:
if the image already has `ctr` the assumption is that the user is specifying an image that already has the things they want to test (like `COS`), so we don't try to do more in this case. (same logic we had in the docker install)

Reply:
ah, I missed this is matching an existing pattern

Reply:
let me change the checks from `ctr` to `containerd`

Reply:
that makes the envvars misleading... if you happen to use an image that already has the binary installed, setting the envvars won't actually cause that version to be used

Reply:
would an approach like this be easier to understand?

```bash
elif [[ "${container_runtime}" == "containerd" ]]; then
  # Install containerd/runc if requested
  if [[ ! -z "${INSTALL_CONTAINERD_VERSION:-}" || ! -z "${INSTALL_RUNC_VERSION}" ]]; then
    install-containerd
  fi
  
  # Verify presence and print versions of ctr, containerd, runc
  if ! command -v ctr >/dev/null 2>&1; then
    echo "ERROR ctr not found. Aborting."
  fi
  ctr --version
  
  if ! command -v containerd >/dev/null 2>&1; then
    echo "ERROR containerd not found. Aborting."
  fi
  containerd --version
  
  if ! command -v runc >/dev/null 2>&1; then
    echo "ERROR runc not found. Aborting."
  fi
  runc --version
fi
```

Reply:
fixing!